classification.yaml

```diff
@@ -0,0 +1,791 @@
+DMS-MWBT-REQ-0030:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Make an empty repo with a POSIX datastore, do get/put.
+DMS-MWBT-REQ-0031:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Make an empty repo with an S3 datastore, do get/put.
+DMS-MWBT-REQ-0028:
+  status: feature-incomplete
+  priority: 1a
+  script:
+    - Make an empty repo with a VOSpace datastore, do get/put.
+  missing:
+    - A VOSpace datastore implementation.
+  notes:
+    Is this actually distinct from S3?
+DMS-MWBT-REQ-0029:
+  status: feature-incomplete
+  priority: 1a
+  script:
+    - Make an empty repo with a VOSpace datastore, do get/put.
+  reuse_script: DMS-MWBT-REQ-0028
+  missing:
+    - A VOSpace datastore implementation.
+  notes:
+    Is this actually distinct from S3?
+DMS-MWBT-REQ-0032:
+  status: unclear
+  priority: 1a
+  notes:
+    Looking at the motivating use case, this requirement seems to really be about supporting both HDF5 and FITS.  But the spec seems to suggest we need to be able to support "any" scientific data format, which is not really enumerable.
+    It also seems to presume that we would have some classes that can be serialized identically to different formats.  But we don't, and I don't really see us adding any just to satisfy this test.
+DMS-MWBT-REQ-0033:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Use a butler to read a subimage via get parameters against any repo.
+DMS-MWBT-REQ-0034:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Use a butler to read a component (e.g. WCS), against any repo.
+DMS-MWBT-REQ-0035:
+  status: unclear
+  priority: 1a
+  notes:
+    This is not a middleware responsibility; it's a feature of whatever in-memory dataset one is trying to operate on (e.g. afw classes).
+DMS-MWBT-REQ-0040:
+  status: feature-incomplete
+  priority: 1a
+  missing:
+    - Tabular-data Datastore, backed by opaque tables.
+  script:
+    - Create or find repo with tabular data in database.
+    - Create or find repo with S3 storage.
+    - Do Butler.get.
+DMS-MWBT-REQ-0045:
+  status: feature-incomplete
+  priority: 1a
+  missing:
+    - Some kind of rendezvous with EFD.
+DMS-MWBT-REQ-0046:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Run ci_hsc_gen3.
+  jira:
+  - LVV-T1982
+  - LVV-T1985
+DMS-MWBT-REQ-0047:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Run ci_hsc_gen3.
+  jira:
+  - LVV-T1982
+DMS-MWBT-REQ-0048:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Make an empty repo.
+    - Ingest some external parquet or FITS catalog.
+    - Call butler.put.
+DMS-MWBT-REQ-0052:
+  status: unclear
+  priority: 1b
+  notes:
+    We aren't planning to need this for batch purposes anymore.  It's still something we could add on top of existing functionality, but it's not so easy to do so that we would want to implement it just for verification.
```

Comment:
I lean towards dropping this requirement as not obviously necessary anymore; it assumes a batch processing architecture different from what we are now building.

Reply:
What might happen is that bps panda plugin is changed to declare the input and output files within rucio and rucio stages the files to the execution node -- then the job itself would use an on-node datastore.